src/amber/server/env.cr

```diff
@@ -1,33 +1,38 @@
 module Amber
   def self.env
-    Env.new(Settings.env.not_nil!.downcase)
+    @@env ||= Env.new(ENV["AMBER_ENV"]? || "development") # Would be best to only define this once.
```

Comment:
I don’t understand the need for memorization here. How much significant is the speed gain?

Reply:
The ENV[“AMBER_ENV”] || “development” this is already happening in Setting.env the reason why it was using the Setting.env. Also initializing a class like this is confusing. What is the use case for Setting.env now with this environment class. 

I would like to recommend to use a variable assignment 
Environment = ENV[“AMBER_ENV”] || “. And remove Settings.env

Reply:
In terms of garbage collection we're potentially saving a lot of time here depending on how many times Amber.env gets called.

Reply:
I think we're on the same page as far as making this the definitely location of env instead of Settings.